.github/workflows/test-python.yaml

```diff
@@ -16,6 +16,20 @@ jobs:
         with:
           python-version: '3.9'
 
+      - name: Install uv and create virtual environment
+        run: |
+          make uv
+          make uv-venv
```

Comment:
This is not needed since the installation is part of `make verify`.
```suggestion
```

Reply:
done